ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_creation.py

```diff
@@ -308,6 +308,38 @@ def test_jax_numpy_uint16(
         )
 
 
+# float16
+@handle_frontend_test(
+    fn_tree="jax.numpy.float16",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+    ),
+)
+def test_jax_numpy_float16(
+    *,
+    dtype_and_x,
+    num_positional_args,
+    as_variable,
+    native_array,
+    on_device,
+    fn_tree,
+    frontend,
+):
+    input_dtype, x = dtype_and_x
+    if ivy.current_backend_str() != "torch":
```

Comment:
This should work for all backends, "torch" included., Why have you added this condition?